community/kernel/src/main/java/org/neo4j/kernel/impl/util/watcher/DefaultFileDeletionEventListener.java

```diff
@@ -39,7 +44,26 @@ public DefaultFileDeletionEventListener( LogService logService )
     @Override
     public void fileDeleted( String fileName )
     {
-        internalLog.info( "Store file '" + fileName + "' was deleted while database was online." );
+        if ( isMonitoredFile( fileName ) )
+        {
+            internalLog.info( format( "Store %s '%s' was deleted while database was running.", getFileType( fileName ),
+                    fileName ) );
+        }
+    }
+
+    private static boolean isMonitoredFile( String fileName )
+    {
+        return !fileName.startsWith( PhysicalLogFile.DEFAULT_NAME );
+    }
+
+    private static String getFileType( String fileName )
+    {
+        return isFile( fileName ) ? "file" : "directory";
+    }
+
+    private static boolean isFile( String fileName )
+    {
+        return fileName.startsWith( MetaDataStore.DEFAULT_NAME ) || fileName.contains( EXTENSION_SEPARATOR );
```

Comment:
I don't think assuming file if file name contains `.` is OK, e.g. don't we use `graph.db` as default directory?